src/MCPClient/lib/clientScripts/normalizeReport.py

```diff
@@ -0,0 +1,248 @@
+    <!doctype html>
+    <html>
+        <head>
+            <meta name="viewport" content="width=device-width">
+            <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+            <title>Normalization failure report: {{ failed_count }} failures found</title>
+        </head>
+        <body>
+            <table cellpadding="10" width="100%">
+                <tr>
+                    <td></td>
+                    <td>
+                        <h1>Archivematica</h1>
+                        <hr />
+                        <h3>
+                            {{ name }}<br />
+                            Normalization failure report: {{ failed_count }} failures found.
+                        </h3>
+                        <table cellpadding="10" border="1" width="100%">
+                            <thead>
+                                <tr>
+                                    <th>File</th>
+                                    <th>Type</th>
+                                    <th>Exit code</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                {% for item in failures %}
+                                    <tr>
+                                        <td><abbr title="{{ item.file_uuid }}">{{ item.location }}</abbr></td>
+                                        {% if item.access_failed %}
+                                            <td>Access</td>
+                                            <td>{{ item.access_task_exit_code }}</td>
+                                        {% endif %}
+                                        {% if item.preservation_failed %}
+                                            <td>Preservation</td>
+                                            <td>{{ item.preservation_task_exit_code }}</td>
+                                        {% endif %}
+                                    </tr>
+                                {% endfor %}
+                            </tbody>
+                        </table>
+                    </td>
+                    <td></td>
+                </tr>
+            </table>
+            <table class="footer-wrap">
+                <tr>
+                    <td></td>
+                    <td class="container">
+                        <div class="content">
+                            <table>
+                                <tr>
+                                    <td align="center">
+                                        <p>Archivematica</p>
+                                    </td>
+                                </tr>
+                            </table>
+                        </div>
+                    </td>
+                    <td></td>
+                </tr>
+            </table>
+        </body>
+    </html>
+"""
+
+
+def query_normalization_results(cursor, uuid):
+    """
+    Execute SQL query on given cursor. Based on https://git.io/vK1Km.
```

Comment:
A path to the source (eg `src/dashboard/src/components/ingest/views_NormalizationReport.py`) or even a github link would be clearer an less likely to suffer from bitrot.